src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -954,7 +971,7 @@
     text: az storage blob upload -f /path/to/file -c MyContainer -n MyBlob
   - name: Upload a file to a storage blob. (autogenerated)
     text: |
-        az storage blob upload --account-name mystorageaccount --account-key 0000-0000 --container-name mycontainer --file /path/to/file --name myblob
+        az storage blob upload --account-key 0000-0000 --account-name mystorageaccount --container-name MyContainer --file /path/to/file --name MyBlob
```

Comment:
reject this one. previous one is right because blob and container should be lower case

Reply:
```suggestion
        az storage blob upload --account-name mystorageaccount --account-key 0000-0000 --container-name mycontainer --file /path/to/file --name myblob
```